src/backend/marsha/urls.py

```diff
@@ -70,7 +71,11 @@
     path("admin/", admin.site.urls),
     # LTI
     path("lti/config.xml", LTIConfigView.as_view(), name="config_lti_view"),
-    path("lti/select/", LTISelectView.as_view(), name="select_lti_view"),
+    re_path(
+        rf"lti/select/((?P<resource_kind>{'|'.join(get_lti_select_resources().keys())})/)?$",
```

Comment:
I think the regex would deserve to be extracted to a variable to ease reading.
Also, maybe add a comment to show how it would be rendered ?

Reply:
I did this in a first time but pylint warn me that I should not do this and change the variable name. I think pylint is applying different rule in this module

Reply:
I found a solution using the constant syntax